src/MaterialTabBar/TabItem.tsx

```diff
@@ -19,6 +21,8 @@ const TabItem: React.FC<MaterialTabItemProps<any>> = ({
   label,
   style,
   labelStyle,
+  activeColor = DEFAULT_ACTIVE_COLOR,
+  inactiveColor = DEFAULT_INACTIVE_COLOR,
```

Comment:
As we are already animating the opacity, we can have only one default color. Please keep both black.